easybuild/easyconfigs/m/MariaDB/MariaDB-10.2.11-foss-2017b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'MariaDB'
+version = '10.2.11'
+
+homepage = 'https://mariadb.org/'
+description = """MariaDB An enhanced, drop-in replacement for MySQL."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['http://ftp.hosteurope.de/mirror/archive.mariadb.org/mariadb-%(version)s/source']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['MariaDB-10.1.13-link-rt-for-jemalloc.patch']
+checksums = [
+    '63555a810db905175a8bd714f6ec77d1b1a11fcfa6d4b97b448cf5bcb6caa3e8',  # mariadb-10.2.11.tar.gz
+    '8295837e623f6c782e1d64b00e0877ea98cce4bf8846755bb86c8a7732797c19',  # MariaDB-10.1.13-link-rt-for-jemalloc.patch
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('ncurses', '6.0'),
+    ('jemalloc', '5.0.1'),
+    ('XZ', '5.2.3'),
+    ('Boost', '1.65.1'),
+    ('libxml2', '2.9.7'),
```

Comment:
@vanzod Looks like this will need to be `2.9.4` to avoid introducing conflicts...